### ISSUE TITLE
Localization does not work with Route::post

### ISSUE THREAD

**Issue by seriousTom** [Waavi/translation#92]
I don`t what is happening but localization does not work with route::post:
```
Route::group(['prefix' => UriLocalizer::localeFromRequest()], function () {
Route::get('/advertisements/save',function(){
		echo trans('main.entry_created');
	})->name('advertisements.store');

Route::post('/advertisements/save',function(){
		echo trans('main.entry_created');
	})->name('advertisements.store');
});
```
With route::get it is always translated correctly, but with route::post it always translates to default locale - english. I have also activated translation middleware globally. Maybe I am using something wrong?

**Comment by lb308** [comment]
the documentation is so hard to follow for me as a newbie
thanks a lot

**Comment by springerin** [comment]
having same issue, specifically with auth...any idea?

**Comment by sildraug** [comment]
If you are using 2.2 this shouldn't be the case. POST requests check both the url AND the session to see what locale to use. I've just added a unit test to check for this.

**Comment by AnasSafi** [comment]
same issue, any idea!

**Comment by MarcelWiebenga** [comment]
I had the same problem. Because this issue isn't fixed yet, I used a simple workaround, maybe somebody else looking for the same issue might find it useful:

In your AppServiceProvider.php place the following inside the boot-function
`App()->setLocale(\UriLocalizer::localeFromRequest());`
